.gitignore

```diff
@@ -176,3 +176,6 @@ python_examples/NZ_example/NZ_inputs/*.csv
 kernprof.py
 *.prof
 *.lprof
+
+# Large csv output files
+large_csv/
```

Comment:
What file is this? I don't think anything should be outputted on the top level anyway - I think we should remove this, and if you want to keep this file locally, then change the path&name to something more descriptive

Reply:
This is the folder holding all the 5.9 million lined csv outputs. Which all are too large to push